pom.xml

```diff
@@ -209,19 +204,18 @@
         <org.snakeyaml.package.import.version.range>[1.17.0,2.0.0)</org.snakeyaml.package.import.version.range>
 
         <!-- Carbon Secure Vault version -->
-        <carbon.config.package.export.version>1.0.0</carbon.config.package.export.version>
-        <carbon.config.package.import.version.range>[1.0.0, 2.0.0)</carbon.config.package.import.version.range>
+        <carbon.config.version>${project.version}</carbon.config.version>
```

Comment:
This pattern (using project.version within properties) normally fails during release plugin release phase. Can we check this with a release dry run?